### PR TITLE
Added -V flag as synonym for --version

### LINKED ISSUE #17: add -v and --version, a la submit50

### PR DESCRIPTION
**Pull request by cmlsharp**
Resolves #17

### CODE REVIEW

File: style50/__main__.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def main():
                         help="specify output mode. Valid modes are character (default), split, unified, score, and json")
     parser.add_argument("-v", "--verbose", action="store_true",
                         help="print full tracebacks of errors")
-    parser.add_argument("--version", action="version", version="%(prog)s {}".format(__version__))
+    parser.add_argument("-V", "--version", action="version", version="%(prog)s {}".format(__version__))
```
Comment (dmalan) [inline style50/__main__.py]: Does this output the program's name? Is that better than just the semver? Seems suboptimal, e.g., that we need to parse astyle's version string.

**Review by dmalan — APPROVED** [review]

File: style50/__main__.py (hunk 1 above)
Comment (cmlsharp) [reply style50/__main__.py]: It does output the program name, yes. What is the connection to `astyle` here? I read in the version number as a tuple and compare that tuple to `(3, 0, 1)` (just like we do for `sys,version_info`)

Also, `__version__` is the standard way to specify version ([PEP8](https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names))

Comment (dmalan) [reply style50/__main__.py]: `astyle -V` outputs `Artistic Style Version 3.0.1`, which seems unnecessarily verbose. And we're currently outputting `style50 x.y.z`, but just `x.y.z` seems sufficient?

Comment (cmlsharp) [reply style50/__main__.py]: Ah sorry I think I understand now! I have no qualms with `style50 -V` just outputting `x.y.z` but it seems fairly conventional for programs to output the name of the program as well (nearly every program I can think of does so).

Comment (cmlsharp) [reply style50/__main__.py]: Not that we have to emulate GNU here but they do standardize what `--version` should output: https://www.gnu.org/prep/standards/html_node/_002d_002dversion.html

Comment (dmalan) [reply style50/__main__.py]: Let's just output `x.y.z` for now. Will potentially be used (more easily) to summarize which versions of CS50 tools a student has installed.

Comment (cmlsharp) [reply style50/__main__.py]: Done

**Pull request merged**